infra/base-images/base-builder/detect_repo_test.py

```diff
@@ -62,6 +62,42 @@ def test_infer_main_repo(self):
       self.check_commit_with_repo(None, None,
                                   'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa', tmp_dir)
 
+  def test_infer_main_repo_from_name(self):
+    """Tests that the main repo can be inferred based on a given repos name."""
```

Comment:
`repo's`
and maybe `from` instead of `based on` and drop `given`

Reply:
Done.